client/analytics/components/report-table/index.js

```diff
@@ -49,6 +53,10 @@ class ReportTable extends Component {
 		}
 	}
 
+	onPageChange() {
+		this.scrollPointRef.current.scrollIntoView();
```

Comment:
I think you need to explicitly pull focus up to the table. when I try to use this with VoiceOver, the screen reader's focus stays on the pagination (it might be different on other screen readers, since I do see that keyboard focus moves to the top)

Reply:
Good catch with that, it should be fixed in the last commits.

Reply:
Hm, now it seems to move me into the search (or download button), which might be confusing. Do you think you can make it focus onto the table? That's already got a tab-stop to it, and it will bring you directly to the content that changed (vs tabbing through the search/actions).

Reply:
That's a good point. I made it so it focuses on `.woocommerce-table__table` in c736abff6e0717b771925556ba4715494c3f3364.

We could also focus directly the first table header or table rows if we want to. I'm not sure what would be more intuitive for users.